enterprise/backend/test/metabase_enterprise/serialization/v2/load_test.clj

```diff
@@ -977,3 +977,37 @@
             (is (some? action))
             (testing ":type should be a keyword again"
               (is (keyword? (:type action))))))))))
+
+(deftest remove-dashcards-test
```

Comment:
You're probably onto this already but you just need an extra test for tabs

Reply:
yep, done